### PR TITLE
feat: redesign ingestion fields

### PR DESCRIPTION
**Pull request by EllaMartirosyan**

### CODE REVIEW

**Review by alebinson — CHANGES_REQUESTED** [review]
gr8!
few changes are needed

File: src/common/components/file-picker/file-picker.tsx [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
+  SupportedLocales
+} from '@map-colonies/react-components';
 import { LayerMetadataMixedUnion } from '../../../discrete-layer/models';
 import CONFIG from '../../config';
 
+const NOT_FOUND = -1;
+const START = 0;
+
+export interface FilePickerComponentHandle {
+  isMultiSelection: () => boolean;
```
[the remark this answers is not in the document]
Comment (EllaMartirosyan) [reply src/common/components/file-picker/file-picker.tsx]: Removed

File: src/discrete-layer/components/dialogs/file-picker-dialog.tsx [hunk 2]
```diff
@@ -4,19 +4,20 @@ import { FormattedMessage } from 'react-intl';
...
-import { Box, FileActionData, FileData, FilePickerActions, FilePickerHandle } from '@map-colonies/react-components';
-import { FilePickerComponent, Selection } from '../../../common/components/file-picker';
+import { Box, FileData } from '@map-colonies/react-components';
+import { FilePickerComponent, FilePickerComponentHandle, Selection } from '../../../common/components/file-picker';
 import { RecordType } from '../../models';
 
 import './file-picker-dialog.css';
 
-const EMPTY = 0;
+// const EMPTY = 0;
```
[the remark this answers is not in the document]
Comment (EllaMartirosyan) [reply src/discrete-layer/components/dialogs/file-picker-dialog.tsx]: Fixed

File: src/discrete-layer/components/dialogs/file-picker-dialog.tsx [hunk 3]
```diff
@@ -115,12 +105,21 @@ export const FilePickerDialogComponent: React.FC<FilePickerDialogComponentProps>
             // @ts-ignore
             ref={filePickerRef}
             files={files}
-            selection={selection} 
-            onFileAction={handleAction}
+            selection={selection}
+            isMultiSelection={recordType !== RecordType.RECORD_3D}
```
[the remark this answers is not in the document]
Comment (EllaMartirosyan) [reply src/discrete-layer/components/dialogs/file-picker-dialog.tsx]: Done

**Review by alebinson — APPROVED** [review]
gr8!

**Pull request merged**